Mathlib/CategoryTheory/Galois/Examples.lean

```diff
@@ -98,6 +99,64 @@ noncomputable instance : FibreFunctor (Action.forget FintypeCat (MonCat.of G)) w
   preservesQuotientsByFiniteGroups _ _ _ := inferInstance
   reflectsIsos := ⟨fun f (h : IsIso f.hom) => inferInstance⟩
 
+instance (X : Action FintypeCat (MonCat.of G)) : MulAction G X.V := Action.instMulAction X
+
+instance : ReflectsIsomorphisms (forget FintypeCat) := by
```

Comment:
This instance should be moved to the file `CategoryTheory.FintypeCat`.

Reply:
I am not sure about this, since this would require importing `RepresentationTheory.Action.Concrete` in `CategoryTheory.FintypeCat`. Do you think this is fine? Maybe its better the other way round? I.e. put it in `RepresentationTheory.Action.Concrete`? There we already have the `FintypeCat` import.

Reply:
I was referring to the `ReflectsIsomorphisms` instance.

Reply:
I replaced it with `Full (forget FintypeCat)` in `CategoryTheory.FintypeCat`.